tests/test_rq_job_utils.py

```diff
@@ -0,0 +1,20 @@
+import pickle
+import zlib
+
+from gsy_e.gsy_e_core.rq_job_handler import launch_simulation_from_rq_job
+
+
+class TestRQJobUtils:
+    """Test RQ-related utilities."""
+    @staticmethod
+    def test_launch_simulation_from_rq_job():
```

Comment:
Sorry I do not see any assertions here, can you help me find what exactly we're testing here

Reply:
It should work without any assertions, but I modified it to explicitly fail in case of a problem.
Thanks

Reply:
I also do not understand how an issue, that is linked in the ticket description will be caught by this test. Can you please explain?

Reply:
The `launch_simulation_from_rq_job` is responsible to start a simulation using the arguments that it receives. The important note is that, if this function fails to start a simulation, then no simulations can be launched from gsy-web.
I felt that writing a simple tests that assures this function runs without any exceptions can be helpful. What do you think?

Reply:
Yeah, I thought the issue was, that there was an unsupported keyword provided gy the gsy-web. But I found the PR that was the origin for this unittest ticket (i think): https://github.com/gridsingularity/gsy-e/pull/1451/files
So indeed, an internal bug in the `launch_simulation_from_rq_job` will be caught by this test. Thanks!